src/explore-education-statistics-admin/src/services/footnoteService.ts

```diff
@@ -2,7 +2,8 @@ import client from '@admin/services/utils/service';
 import footnoteToFlatFootnote from './utils/footnote/footnoteToFlatFootnote';
 
 export interface FootnoteSubject {
-  selected: boolean;
+  selectionType: string;
+  selected?: boolean;
```

Comment:
Do we use the `selected` flag anymore, now we have `selectionType`?

Reply:
Created a ticket to address this: EES-2038